Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:latest
```

Comment:
Could you please try to change this to Alpine.
Alpine is very small linux distro on has docker and very less amount of boot time.
So, we better use that one.

Reply:
ok will change it